src/Cairo.jl

```diff
@@ -97,6 +97,36 @@ else
     get_stream_callback(T) = cfunction(write_to_stream_callback, Int32, (Ref{T}, Ptr{UInt8}, UInt32))
 end
 
+
+function read_from_stream_callback(s::IO, buf::Ptr{UInt8}, len::UInt32)
+    # wrap the provided buf into a julia Array
+
+    if VERSION < v"0.5.0-rc"
+        b1 = pointer_to_array(buf,len); #pre-v0.5
+    else
+        b1 = unsafe_wrap(Array{UInt8,1},buf,len);
```

Comment:
This version is supported by Compat, so I think you don't need the VERSION check. https://github.com/JuliaLang/Compat.jl#renamed-functions